tvm/src/codegen/hlsc/codegen_shls.h

```diff
@@ -0,0 +1,124 @@
+/*!
+ * Copyright (c) 2021 by Contributors
+ * \file codegen_vhls.h
+ * \brief Generate Stratus HLS kernel code.
+ */
+#ifndef CODEGEN_HLSC_CODEGEN_SHLS_H_
+#define CODEGEN_HLSC_CODEGEN_SHLS_H_
+
+#include <tvm/codegen.h>
+#include <tvm/packed_func_ext.h>
+#include <fstream>
+#include <string>
+#include "./codegen_hlsc.h"
+#include "./codegen_vhls.h"
+
+namespace TVM {
+namespace codegen {
+
+class CodeGenStratusHLS final : public CodeGenVivadoHLS {
```

Comment:
Actually this class inherence relation is weird. I suppose Vivado HLS and Stratus HLS should be at the same level of abstraction? Otherwise, this line of code indicates Stratus HLS can be viewed as another kind of Vivado HLS.